backend/app/src/main/resources/templates/admin/profanity.html

```
@@ -0,0 +1,363 @@
+
+        <div class="form-card">
+            <form th:action="@{/admin/profanity/words}" method="post">
+                <input type="hidden" name="search" th:value="${search}">
+                <input type="hidden" name="language" th:value="${language}">
+                <input type="hidden" name="source" th:value="${source}">
+                <input type="hidden" name="activeFilter" th:value="${activeFilter}">
+                <input type="hidden" name="wordsPage" th:value="${wordsPage}">
+                <div class="form-row">
+                    <div class="form-field">
+                        <label class="form-label">단어</label>
+                        <input class="form-input" type="text" name="word" required maxlength="200"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)